include/envoy/server/instance.h

```diff
@@ -241,6 +242,17 @@ class Instance {
    * @return Configuration::TransportSocketFactoryContext& factory context for transport sockets.
    */
   virtual Configuration::TransportSocketFactoryContext& transportSocketFactoryContext() PURE;
+
+  /**
+   * Set the default server-wide tracer provider configuration that will be used as a fallback
+   * if an `envoy.http_connection_manager` filter that has tracing enabled doesn't define
```

Comment:
nit: replace "envoy.http_connection_manager" with the non-deprecated name?

Reply:
fixed